delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/model/FileCommand.scala

```diff
@@ -64,6 +65,7 @@ object FileCommand {
       storage: ResourceRef.Revision,
       storageType: StorageType,
       attributes: FileAttributes,
+      userMetadata: Option[FileUserMetadata],
```

Comment:
We could add more fields to `FileAttributes` ?
It would allow to get rid of the metadata confusion.
In all cases, we should avoid to import something from the elasticsearch plugin here

Reply:
I can't tell whether that's an improvement. All of those things are calculated, where this is user-supplied information. It feels like it might be better to keep them separate

Reply:
They can stay separated but it would be nice to have a naming involving the metadata term to avoid confusion